Components/Pages/editGoal.js

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import { useState, useEffect } from "react";
+import CreateContainer from "../Molecules/createContainer";
+import {
+  View,
+  StyleSheet,
+  Dimensions,
+  Linking,
+  FlatList,
+  TouchableOpacity,
+  Image,
+  TextInput,
+} from "react-native";
+
+import Constants from "expo-constants";
+import { TextField } from "../Atoms/Fields";
+import { Dropdown } from "react-native-material-dropdown";
+import { useGoalUpdate, useGoalsPull } from "../../API";
+
+const editGoal = ({ navigation }) => {
+  const { goals, refetch } = useGoalsPull();
+  const { updateGoal } = useGoalUpdate();
+
+  const _id = navigation.getParam("_id");
+  refetch();
```

Comment:
useEffect hook can be used here too?

Reply:
great suggestion - anywhere refetch is, we'll put in useEffect